src/types/external/duration.rs

```diff
@@ -9,7 +9,7 @@ use crate::{InputValueError, InputValueResult, Scalar, ScalarType, Value};
 /// The input/output is a string in ISO8601 format.
 #[Scalar(
     internal,
-    name = "Duration",
+    name = "ChronoDuration",
```

Comment:
Can we keep this scalar name as `Duration` or will it collide with `jiff`'s version?

The reason I ask is because if these are meant to be ISO 8601 durations, then the backing implementation (whether `chrono` or `jiff`) shouldn't actually matter to a GraphQL consumer. Also swapping one backend for another shouldn't require churning the schemas (assuming they are being either committed somewhere or the consumers are using something like Apollo Client for code generation)